### ISSUE TITLE
Does this not work on windows?

### ISSUE THREAD

**Issue by RyanConway91** [pymt-lab/pymt_prms_surface#9]
I believe I have gone through the installation steps correctly. Everything works up to:
conda install bmi-fortran=2.0 prms prms_surface
Then I get PackagesNotFound error from conda.

**Comment by mdpiper** [comment]
@RyanConway91 You're right--I only built `conda` binaries for *prms*, *prms_surface* and *pymt_prms_surface* on Linux and macOS.

~There's no reason `conda` binaries can't be built on Windows for these packages, as well. We just didn't have the funding to do so.~ (see below)

**Comment by rmcd-mscb** [comment]
Thanks @mdpiper, Also, if I remember correctly, compiling prms (FORTRAN) requires netCDF.  Windows FORTRAN netCDF bindings are not well supported.  If you want to give the prms BMIs a try, you could run them on WSL2, on windows.

**Comment by mdpiper** [comment]
@rmcd-mscb Oh, that's right! Thank you. I'd forgotten abou the netcdf-fortran issue on Windows.
